.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -9,16 +9,22 @@ assignees: ''
 # Onboarding
 
 By creating this ticket, you are requesting to be onboarded as a developer on the Azul team.
-The action items in this ticket will be carried out by existing team members.
-Your next step is to carefully read the following documents:
+The action items in this ticket will be carried out by you and existing team
+members.
+
+Your next steps are to carefully read the following documents:
 
 - [ ] [Azul README.md](https://github.com/databiosphere/azul)
 
 - [ ] [Contributing Guidelines](https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst)
 
 - [ ] [AWS Credential Configuration](https://giwiki.gi.ucsc.edu/index.php/Overview_of_Getting_and_Using_an_AWS_IAM_Account)
 
-Provide access to cloud infrastructure and tools:
+Install the following prerequisites:
+
+- [ ] [git-secrets](https://github.com/databiosphere/azul#211-git-secrets)
```

Comment:
Relative link, please, so 1) we don't have to adjust them when the repo moves or is renamed and 2) we can test the links in this PR. While you're at it, please convert the other links here, too.